src/test/java/com/cicsok/sekta/web/rest/controller/DummyRESTControllerIT.java

```diff
@@ -0,0 +1,51 @@
+package com.cicsok.sekta.web.rest.controller;
+
+import com.cicsok.sekta.web.rest.marker.ControllerIntegrationTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.function.Supplier;
+
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@ControllerIntegrationTest
+public class DummyRESTControllerIT {
```

Comment:
We should not see Java class changes under this PR.
Could you please perform a rebate operation?